.github/workflows/deploy_staging.yml

```
@@ -21,11 +21,36 @@ jobs:
       ecs-cluster-name: "isomer-stg-ecs"
       ecs-web-service-name: "isomer-stg-ecs-service"
       ecs-container-name: "backend"
+      ecs-container-port: 8081 
       environment: "staging"
       shortEnv: "stg"
       task-definition-path: ".aws/deploy/backend-task-definition.staging.json" 
       codedeploy-application: "isomer-stg-ecs-app"
       codedeploy-deployment-group: "isomer-stg-ecs-dg"
+      path-to-dockerfile: "Dockerfile"
+
+    secrets: 
+      AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID }}
+      EFS_FILE_SYSTEM_ID: ${{ secrets.STAGING_EFS_FILE_SYSTEM_ID }}
+      DD_API_KEY: ${{ secrets.DD_API_KEY }}
+  
+  deploy-support: 
+    name: Deploy support to staging
+    uses: ./.github/workflows/aws_deploy.yml
+    with:
+      aws-region: "ap-southeast-1"
+      cicd-role: "arn:aws:iam::095733531422:role/isomer-infra-github-oidc-role-16ea937"
+      ecr-repository: "isomer-infra-staging-ecr"
+      ecs-cluster-name: "isomer-stg-ecs"
+      ecs-web-service-name: "stg-support-ecs-service"
```

Naming convention doens't seem to match the deploy-app name? Did you mean
```suggestion
      ecs-web-service-name: "isomer-support-stg-ecs-service"
```